src/demo/pages/ButtonPage/examples/Button.Basic.Example.tsx

```diff
@@ -44,6 +44,12 @@ export class ButtonBasicExample extends React.Component<any, IBasicButtonsExampl
         <Label>Command button</Label>
         <Button disabled={ disabled } buttonType={ ButtonType.command } icon='personAdd' description='Description of the action this button takes'>Create account</Button>
 
+        <Label>Primary Button with Icon</Label>
+        <Button disabled={ disabled } buttonType={ ButtonType.primary } title='Star' description='Description of the action this button takes'>
```

Comment:
you could add a classname to this button if you'd like, e.g. 'ms-ButtonBasicExample-customButton'

then your scoped css would be:

.ms-ButtonBasicExample-customButton .ms-Icon {
...
}